src/com/centeksoftware/parclock/javarelay/MainFrame.java

```diff
@@ -247,6 +268,23 @@ private void initComponents()
 		action1 = new javax.swing.JLabel();
 		action2 = new javax.swing.JLabel();
 		
+		SimpleDateFormat sdfDate = new SimpleDateFormat("yyyy-MM-dd_HH;mm;ss");// dd/MM/yyyy
+		Date now = new Date();
+		String timestamp = sdfDate.format(now);
+		timestamp = timestamp.replace(" ", "_");
```

Comment:
Why?

Reply:
When I googled how to get a string of the current date/time this is what I found, excluding line 274, which I now realize is redundant. Is their something simpler I should be doing instead?

Reply:
Actually given the other two comments it seems like you are making this statements in regard to line 274, which I have now removed and will be sending a new PR with the other changes